deployments/oathkeeper/rules/rules.yaml

```diff
@@ -0,0 +1,39 @@
+- id: workday-everything
+  upstream:
+    url: https://host.docker.internal:8080
+  version: v0.40.3
+  match:
+    url: https://workday.flock.community/<.*>
```

Comment:
We need the upstream url of workday, which is the default url it gets assigned when the google cloud run was created. Something like: `https://workday-app-tveu425laa-ew.a.run.app`

Reply:
Updated with `https://workday-app-x6nsuevzyq-ew.a.run.app/`